src/app/error.tsx

```diff
@@ -14,47 +13,45 @@ export default function Error({
   console.log("Unhandled error: ", error);
 
   return (
-    <LayoutContentContainer>
-      <Card>
-        <Box gap="xl" align="start">
-          <Box gap="md">
-            <Heading as="h2" size="xs" weight="medium">
-              Unhandled Error
-            </Heading>
+    <Card>
+      <Box gap="xl" align="start">
+        <Box gap="md">
+          <Heading as="h2" size="xs" weight="medium">
+            Unhandled Error
+          </Heading>
 
-            <Text size="sm" as="p">
-              Uh-oh, we didn’t handle this error. We would appreciate it if you
-              opened an issue on GitHub, providing as many details as possible
-              to help us fix this bug.
-            </Text>
-          </Box>
+          <Text size="sm" as="p">
+            Uh-oh, we didn’t handle this error. We would appreciate it if you
+            opened an issue on GitHub, providing as many details as possible to
```

Comment:
it'd be nice to have a link to an issue!

Reply:
We can add a link to the issue, but looking at Sentry logs might be more helpful (once we have it in `main`).